k8s/src/test/scala/io/buoyant/k8s/ServiceNamerTest.scala

```diff
@@ -32,7 +32,7 @@ class ServiceNamerTest extends FunSuite with Awaits {
         rsp.content = Rsps.Init
         Future.value(rsp)
 
-      case req if req.uri == "/api/v1/namespaces/pythonsky/services?watch=true&resourceVersion=11600623" =>
+      case req if req.uri == "/api/v1/watch/namespaces/pythonsky/services?resourceVersion=11600623" =>
```

Comment:
I think we need to make the same update in the IngressIdentifierTest.scala test file as well, here:

https://github.com/linkerd/linkerd/blob/eliza/configmap-interpreter/linkerd/protocol/h2/src/test/scala/io/buoyant/linkerd/protocol/h2/IngressIdentifierTest.scala#L55